database/README.md

```diff
@@ -398,3 +398,24 @@ hasura metadata apply --envfile .env.local
 ```
 
 Once we see that no errors occur when applying the sequence of migrations locally, we can merge and the CI will apply the new migrations and metadata to the staging database.
+
+## Database Schema Documentation
+
+CI exists to automatically generate a database schema documentation file. This occurs
+when a PR is created and subsequently when a commit is pushed onto a PR'd branch.
+The CI is performed by a GitHub action which does the following:
+
+1. Install the Hasura CLI for `graphql-engine`.
+2. Spin up a postgres database which is initially empty.
+3. Use the `hasura` CLI to deploy migrations, which build up the VZ DB
+4. Install and use the `dbdocs` npm tool to generate a DBML file for the DB
+5. Use teh `dbdocs` tool again to upload the documentation to https://dbdocs.io.
```

Comment:
small typo teh/the

Reply:
Thanks - great catch! f552d2f8ea3bb7374907a59688dde4c84f44db71